webSites/webSites-war/src/main/java/org/silverpeas/components/websites/servlets/WebSitesRequestRouter.java

```diff
@@ -23,38 +23,35 @@
  */
 package org.silverpeas.components.websites.servlets;
 
+import org.apache.commons.fileupload.FileItem;
+import org.silverpeas.components.websites.control.WebSiteSessionController;
+import org.silverpeas.components.websites.service.WebSitesException;
 import org.silverpeas.components.websites.siteManage.model.FolderDetail;
-import org.silverpeas.core.util.URLUtil;
+import org.silverpeas.components.websites.siteManage.model.SiteDetail;
+import org.silverpeas.core.contribution.publication.model.PublicationDetail;
+import org.silverpeas.core.node.model.NodeDetail;
+import org.silverpeas.core.node.model.NodePK;
 import org.silverpeas.core.util.EncodeHelper;
+import org.silverpeas.core.util.ResourceLocator;
 import org.silverpeas.core.util.SettingBundle;
 import org.silverpeas.core.util.StringUtil;
+import org.silverpeas.core.util.URLUtil;
 import org.silverpeas.core.util.file.FileUploadUtil;
+import org.silverpeas.core.util.logging.SilverLogger;
+import org.silverpeas.core.web.http.HttpRequest;
 import org.silverpeas.core.web.mvc.controller.ComponentContext;
 import org.silverpeas.core.web.mvc.controller.MainSessionController;
 import org.silverpeas.core.web.mvc.route.ComponentRequestRouter;
-import org.silverpeas.core.silvertrace.SilverTrace;
-import org.silverpeas.core.util.ResourceLocator;
-import org.silverpeas.core.node.model.NodeDetail;
-import org.silverpeas.core.node.model.NodePK;
-import org.silverpeas.core.contribution.publication.model.PublicationDetail;
-import org.silverpeas.components.websites.control.WebSiteSessionController;
-import org.silverpeas.components.websites.siteManage.model.SiteDetail;
 
+import javax.servlet.http.HttpServletRequest;
 import java.io.File;
+import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Iterator;
 import java.util.List;
-import javax.servlet.http.HttpServletRequest;
-
-import org.apache.commons.fileupload.FileItem;
-import org.silverpeas.core.web.http.HttpRequest;
 
-/**
- * Class declaration
- * @author
- */
 public class WebSitesRequestRouter extends ComponentRequestRouter<WebSiteSessionController> {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this class is 201 which is greater than 200 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AClassCyclomaticComplexity)